benchmarks/benchmark_serving.py

```diff
@@ -185,6 +184,29 @@ def sample_sonnet_requests(
     return sampled_requests
 
 
+def sample_random_requests(input_len, output_len, num_prompts, range_ratio,
```

Comment:
Please add type hint for parameters in the function signature

Reply:
Just fixed, please check it again

Reply:
Update the formatting issues for you. For future reference, you can run the `format.sh` we provide in the repo for easily formatting your code changes.

Reply:
OK, I will. Thinks for the tip